src/Dnsimple/Client.php

```diff
@@ -22,21 +43,86 @@ class Client
     const API_VERSION = "v2";
 
     /**
-     * The default base URL for API requests.
+     * URL to the production environment
      */
-    const BASE_URL = "https://api.sandbox.dnsimple.com";
+    const BASE_URL = "https://api.dnsimple.com";
+    const DEFAULT_USER_AGENT = "dnsimple-php/" . VERSION;
 
 
     /**
      * @var string $accessToken the bearer authentication token
      */
-    private $accessToken;
+    private string $accessToken;
+
+    /**
+     * The http client we are using to send requests to the DNSimple API.
+     *
+     * @var GuzzleHttp\Client $httpClient the HTTP client
+     */
+    private GuzzleHttp\Client $httpClient;
 
     /**
-     * @var \GuzzleHttp\Client $httpClient the HTTP client
+     * @var Identity The service handling the Identity API
+     */
+    public Identity $Identity;
+    /**
+     * @var Accounts The service handling the Accounts API
+     */
+    public Accounts $Accounts;
+    /**
+     * @var Domains The service handling the Domains API
+     */
+    public Domains $Domains;
+    /**
+     * @var Registrar The service handling the Registrar API
+     */
+    public Registrar $Registrar;
+    /**
+     * @var Zones The service handling the Zones API
+     */
+    public Zones $Zones;
+    /**
+     * @var Oauth The service handling the Oauth API
+     */
+    public Oauth $Oauth;
+    /**
+     * @var Tlds The service handling the Tlds API
+     */
+    public Tlds $Tlds;
+    /**
+     * @var Services The service handling the Services API
+     */
+    public Services $Services;
+    /**
+     * @var Templates The service handling the Templates API
+     */
+    public Templates $Templates;
+    /**
+     * @var VanityNameServers The service handling the Vanity Name Servers API
+     */
+    public VanityNameServers $VanityNameServers;
+    /**
+     * @var Webhooks The service handling the Webhooks API
+     */
+    public Webhooks $Webhooks;
+    private string $customUserAgent = "";
```

Comment:
What about moving this definition closer to `$accessToken`? It seems a bit weird to have it dumped in the middle of the service lists (especially given it's private, and it's missing the docs).